src/main/java/core/basesyntax/strategy/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.TransactionDto;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    @Override
+    public void apply(TransactionDto transactionDto) {
+        //validate: do we have enough
+    }
+}

```

Comment:
Implement this method, create input.csv with data and empty output.csv. Try to run the program and see what will be written in the output.csv